sdkcompat/v232/BUILD

```diff
@@ -0,0 +1,42 @@
+# Description: Indirections for SDK changes to the underlying platform library.
+
+load("//intellij_platform_sdk:build_defs.bzl", "select_for_ide")
+
+licenses(["notice"])
+
+java_library(
+    name = "v232",
+    srcs = glob([
+        "com/google/idea/sdkcompat/general/**",
+        "com/google/idea/sdkcompat/logging/**",
```

Comment:
I think you need to copy `com/google/idea/sdkcompat/vcs` from sdkcompat of v231 and add it to this `glob` to make future cherry-picks from google branch easier.

Reply:
I see that you updated it but can you please copy the folder [v231/com/google/idea/sdkcompat/vcs](https://github.com/bazelbuild/intellij/tree/master/sdkcompat/v231/com/google/idea/sdkcompat/vcs) to v232 sdkcompat?